### PR TITLE
Create invest guide -  index.md

### PR DESCRIPTION
**Pull request by sirjared**

### CODE REVIEW

File: docs/use/Invest/index.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    - [Talisman - Polkadot Wallet](https://chrome.google.com/webstore/detail/talisman-polkadot-wallet/fijngjgcjhjmmpcmkeiomlglpeiijkld)
+    - [SubWallet](https://chrome.google.com/webstore/detail/subwallet-polkadot-extens/onhogfjeacnfoofkfgppdlbmlmnplgbn)
+- Ethereum
+    - [Ledger](https://www.ledger.com/) (Note, if using a Ledger Device, please enable "Blind Signing" in the Ethereum app settings and ensure the device has been updated to the latest firmware. It's recommended to disable blind signing after each use.)
+    - [Metamask](https://metamask.io/download.html)
+    - [Wallet Connect](https://walletconnect.com/)
+
+
+If you are setting up a wallet for the first time please apply appropriate security measures when setting up your web3 wallet. Find more information how to set up a web3 wallet [here](/use/setup-wallet/).
+
```
Comment (sirjared) [inline docs/use/Invest/index.md:32]: What link should go in 31

File: docs/use/Invest/index.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
+---
+id: invest
+order: 3
+title: Investing into and Redeeming from a pool
+contributors: <Dennis Wellmann:dennis@k-f.co>, <Jay:jay@k-f.co>
+---
+
```
Comment (sirjared) [inline docs/use/Invest/index.md:7]: Please adjust image size as needed, i used 600px to start for all of them.

File: docs/use/Invest/index.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+# Investment requirements
+
+## Legal requirements
+- KYC/KYB has been successfully completed
+    - Find more information how to onboard [here](/use/onboarding/)
```
Comment (sirjared) [inline docs/use/Invest/index.md:12]: What link should go here?

File: docs/use/Invest/index.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+Once you are verified and completed pool onboarding by signing the subscription agreement, uploading tax documents, and for US investors certifying investor Accreditation status, you can begin investing in the pool.
+
+- Select the Pool you would like to invest in.
+- From the drop down menu Choose the Pool tranche you want to or have already completed onboarding for. When onboarding you will need to confirm KYC, sign the pools Subscription agreement, and provide Tax documentation. ![](./images/select_tranche.png#width=600px) 
+- Enter the Amount you would like to invest. Please note each pool may have a minimum Investment amount, in the example pool seen above, the minimum is 10,000 aUSD. Beneath this number, you can also see your wallet's balance of the stablecoin. 
+- You can also click the MAX button to invest your entire Available Balance. 
+- Once you enter the amount, Click "Invest" and sign the On-chain transaction. Note, once you have clicked "Invest" your order is locked but not Executed into the pool. You must wait for the epoch to conclude before your Pending investment is executed. [Learn more about Epochs and Pending orders Here](https://).
```
Comment (sirjared) [inline docs/use/Invest/index.md:88]: Please include proper link here

File: docs/use/Invest/index.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+![](./images/change_order_value.png#width=600px)
+
+
+
+### How to check investment/redemption status
+- You can check the status of your Pending Order by navigating to "Liquidity" tab of the pool and seeing the remaining time on the epoch for orders to execute.
+
+![](./images/check_status.png#width=600px)
+
+- Scroll down to the bottom of the page to view Order Overview. You can learn more about how orders are processed [here](https://docs.centrifuge.io/learn/epoch/).
```
Comment (sirjared) [inline docs/use/Invest/index.md:104]: Please check link is correct

**Review by denniswell — APPROVED** [review]

**Pull request merged**